src/main/java/org/jenkinsci/plugins/docker/traceability/model/DockerTraceabilityReport.java

```diff
@@ -70,7 +71,7 @@ public DockerTraceabilityReport() {
     
     public DockerTraceabilityReport(@Nonnull Event event, @Nonnull Info hostInfo, 
             @CheckForNull InspectContainerResponse container, 
```

Comment:
if the container no longer exists it could be `null`. it's already required=false so should be `@CheckForNull` also.